src/ucp/core/ucp_ep.c

```diff
@@ -956,6 +956,19 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
         }
     }
 
+    /* Configuration for memory domains */
+    config->dn = ucs_malloc(sizeof(ucp_ep_dn_config_t) * context->num_mds, "domain-specific configuration");
```

Comment:
is it context->num_mds? or num mds with UCT_MD_FLAG_ADDR_DN?

Reply:
Yeah later is less size, I will modify this